frontend/packages/console-shared/src/hooks/useClusterToken.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { consoleFetchJSON as coFetchJSON } from '@console/dynamic-plugin-sdk/src/utils/fetch';
+import { useActiveCluster } from '@console/shared/src/hooks/useActiveCluster';
+
+export const useClusterToken = (): [string] => {
```

Comment:
I'd suggest `useRequestTokenURL` or similar since we're not returning the actual token, just the URL.